extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.js

```diff
@@ -241,7 +242,8 @@ export class AmpLightboxGallery extends AMP.BaseElement {
         sourceElement: element,
       };
       let slide = clonedNode;
-      if (clonedNode.tagName === 'AMP-IMG') {
+      const tagName = clonedNode.tagName.toLowerCase();
```

Comment:
I see you're calling `tagName.toLowerCase()` a lot. You could save some bytes if you use uppercase tag names for the keys in `ELIGIBLE_TAP_TAGS` so you don't have to convert.

Reply:
Yeah this is a much better idea lol.